src/standalone/layout.jsx

```diff
@@ -61,7 +61,7 @@ export default class StandaloneLayout extends React.Component {
         { !loadingStatus || loadingStatus === "success" && <BaseLayout /> }
         <Row>
           <Col>
-            <OnlineValidatorBadge />
+            {/* <OnlineValidatorBadge /> */}
```

Comment:
Do we still want this commented out here? If so, can you please add a note as to why it was commented out, and why you'd want to uncomment it?

Reply:
this validates the swagger.json and adds a badge indicating if the spec file is compliant. right now it always produces errors with out swagger files. could also be re-added back in if the generated swagger docs are fixed to be compliant with the validator.